server/odc-server/src/main/java/com/oceanbase/odc/server/OdcServer.java

```diff
@@ -89,10 +89,10 @@ public static void main(String[] args) {
             log.info("Task executor exit.");
             return;
         }
-        AlarmUtils.alarm(SERVER_RESTART, LocalDateTime.now().toString());
         initEnv();
         System.setProperty("spring.cloud.bootstrap.enabled", "true");
         PluginSpringApplication.run(OdcServer.class, args);
+        AlarmUtils.alarm(SERVER_RESTART, LocalDateTime.now().toString());
```

Comment:
this statement may never be run?
odc seems to be hang up at `PluginSpringApplication.run(OdcServer.class, args);`

Reply:
will execute if start
![image](https://github.com/user-attachments/assets/a1ed0c22-5386-4ac9-906a-166db6538f9f)